src/content/ja/2021/seo.md

```diff
@@ -0,0 +1,758 @@
+- デスクトップ用ページの中央値には21個の`<img>`タグがあり、モバイル用ページの中央値には19個の`<img>`タグがあります。
+- デスクトップページの10%は、83個以上の`<img>`タグを持っています。モバイルページの10%は、73個以上の`<img>`タグを持っています。
+
+この数字は2020年以降、ほとんど変化していません。
+
+#### 画像の `alt` 属性
+
+`<img>` 要素の `alt` 属性は、画像の内容を説明するのに役立ち、<a hreflang="en" href="https://almanac.httparchive.org/en/2021/accessibility">アクセシビリティ</a>へ影響を及ぼします。
+
+なお、`alt`属性の欠落は問題を示さない場合があります。ページには極端に小さい画像や空白の画像が含まれていることがありますが、SEO（あるいはアクセシビリティ）上の理由から `alt` 属性は必要でありません。
+
+{{ figure_markup(
+   image="images-with-alt-attribute.png",
+   caption="`alt` 属性を含む画像の割合。",
+   description="パーセンタイル（10、25、50、75、90）ごとに`alt`属性が存在する画像の数を示す棒グラフです。このデータから、中央のウェブページには、モバイルページで54.6%、デスクトップページでは56.5%の`alt`属性の付いた画像が含まれていることがわかりました。",
+   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQ7oSHa6DHYTfZqsuGmIYdlfYVkwrUvOYD_r6soecExV_ZpbbZjmG6watu0hwrOKqK3-inNrt0TfXCO/pubchart?oid=1862003290&format=interactive",
+   sheets_gid="412947118",
+   sql_file="image-alt-stats.sql"
+   )
+}}
+
+{{ figure_markup(
+   image="images-with-blank-alt-attribute.png",
+   caption="`alt` 属性が空白の割合。",
+   description="パーセンタイル（25、50、75、90）ごとに、空白`alt`属性がフィーチャーされる割合を示す棒グラフ。中央のウェブページでは、デスクトップで10.5%、モバイルでは11.8%の空白の`alt`属性が表示されています。",
+   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQ7oSHa6DHYTfZqsuGmIYdlfYVkwrUvOYD_r6soecExV_ZpbbZjmG6watu0hwrOKqK3-inNrt0TfXCO/pubchart?oid=198831003&format=interactive",
+   sheets_gid="412947118",
+   sql_file="image-alt-stats.sql"
+   )
+}}
+
+{{ figure_markup(
+   image="images-with-missing-alt-attribute.png",
+   caption="`alt`属性がない画像の割合。",
+   description="パーセンタイル（10、25、50、75、90）ごとに`alt`属性が欠落している画像の割合を示す棒グラフです。中央のウェブページでは、デスクトップでは `alt` 属性の欠落が1.4%、モバイルでは `alt` 属性の欠落が0%でした。",
+   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQ7oSHa6DHYTfZqsuGmIYdlfYVkwrUvOYD_r6soecExV_ZpbbZjmG6watu0hwrOKqK3-inNrt0TfXCO/pubchart?oid=819909313&format=interactive",
+   sheets_gid="412947118",
+   sql_file="image-alt-stats.sql"
+   )
+}}
+
+私たちはそれがわかりました
+
+- デスクトップ用ページの中央値では、56.5%の `<img>` タグに `alt` 属性が設定されています。これは、2020年に比べてわずかに増加しています。
+- モバイルページの中央値では、54.6%の `<img>` タグに `alt` 属性が設定されています。これは、2020年と比較して若干の増加です。
+- しかし、デスクトップとモバイルページの中央値では、10.5%と11.8%の `<img>` タグが `alt` 属性を空白にしています（それぞれ）。これは事実上、2020年と同じです。
+- 中央のデスクトップとモバイルのページでは、`<img>` タグに `alt` 属性なしがゼロか、それと近い値になっています。これは、中央ページにある`<img>`タグの2～3%に`alt`属性がなかった2020年に比べて改善されています。
+
+#### 画像 `loading` 属性
+
+`<img>` 要素の `loading` 属性は、ユーザーエージェントがページ上の画像のレンダリングと表示をどのように優先させるかに影響します。ユーザー体験やページの読み込み性能に影響を与える可能性があり、これらはいずれもSEOの成功に影響を与えます。
+
+{{ figure_markup(
+   image="image-loading-property-usage.png",
+   caption="画像読み込みのプロパティの使用状況。",
+   description='ページと画像読み込みプロパティの使用率（missing, lazy, eager, invalid, auto, blank）を示す棒グラフ。当社のデータでは、デスクトップページの83.3%、モバイルページの83.5%で画像の読み込みプロパティが欠落していることが判明しました。その結果、デスクトップとモバイルのページの15.6%が `loading="lazy"` を使用しており、`loading="eager"` はデスクトップとモバイルのページの0.8%に過ぎないことがわかりました。その他の画像読み込みプロパティである `loading="invalid"` や `loading="auto"` 、 `loading="blank"` は、デスクトップやモバイルのページの1%未満にしか使用されていません。',
```

Comment:
Can you correct this, following the recent correct in https://github.com/HTTPArchive/almanac.httparchive.org/pull/2788/files ?

Reply:
@tunetheweb Sorry, I don't understand.
What do I need to fix?

Reply:
Oops. Wrong link: https://github.com/HTTPArchive/almanac.httparchive.org/pull/2787